include/grpc/grpc.h

```diff
@@ -127,6 +127,11 @@ typedef struct {
 /** Initial sequence number for http2 transports */
 #define GRPC_ARG_HTTP2_INITIAL_SEQUENCE_NUMBER \
   "grpc.http2.initial_sequence_number"
+/** How much memory to use for hpack decoding */
+#define GRPC_ARG_HTTP2_HPACK_TABLE_SIZE_DECODER \
+  "grpc.http2.hpack_table_size.decoder"
+#define GRPC_ARG_HTTP2_HPACK_TABLE_SIZE_ENCODER \
```

Comment:
super nit: have the corresponding /** How much memory to use for hpack encoding */ for GRPC_ARG_HTTP2_HPACK_TABLE_SIZE_ENCODER, for consistency, given that grpc.h is so visible.